### PR TITLE
develop merge

### PR DESCRIPTION
**Pull request by alan-ls**
I've added comments to the 3 places where there was conflicts.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,4 +105,50 @@ run-edge1:
 # - SENDGRID_SENDER
 bridgerun:
 	${REBAR} as bridge release
-	_build/bridge/rel/bondy/bin/bondy console
\ No newline at end of file
+	_build/bridge/rel/bondy/bin/bondy console
```
Comment (alan-ls) [inline Makefile:108]: This was on conflict

File: rebar.lock [hunk 2]
```diff
@@ -28,18 +28,14 @@
...
-       {ref,"002e6baa58923f02a98a362650a336ea9e18e06f"}},
+       {ref,"f7339ad2a57537b3f7e185a38341664d3cc22cfb"}},
   1},
  {<<"email">>,
   {git,"https://github.com/kivra/email.git",
        {ref,"b62e070111e635e739628254eab8db2835631d28"}},
   0},
  {<<"enacl">>,{pkg,<<"enacl">>,<<"1.2.1">>},0},
  {<<"erlcloud">>,{pkg,<<"erlcloud">>,<<"3.5.10">>},0},
- {<<"eunit_formatters">>,
```
Comment (alan-ls) [inline rebar.lock]: This was a conflict

**Comment by aramallo** [comment]
@alan-ls you've included more than 280 files as you have an old version which contained a commit of the docs....can you cleanup and only submit the changes you want to take effect?

**Comment by alan-ls** [comment]
@aramallo I've just blindly merged the latest at the time: commit 9e30552f from Jan 21 2022.
My git logs show all my remotes point to the same commit. `upstream` refers to `leapsight/bondy` while `origin` refers to my fork `alan-ls/bondy`.
```
% git log --oneline --graph
*   c494ff82 (HEAD -> feature/amiguez.notification_broker_bridges, origin/feature/amiguez.notification_broker_bridges) Merge remote-tracking branch 'origin/develop' into feature/amiguez.notification_broker_bridges
|\  
| * 9e30552f (upstream/develop, origin/develop, origin/HEAD) Upgrade to plum_db with fix to recent bug that prevents ram_disk prefixes to be loaded into ets memory tables
| * 7bccf320 fix process registration
...
```

I had a closer look at the changes and, in terms of number of affected files, most of them come from 2 changes:

1. Copyright updates (127 files touched at 3d03f6a)
2. Deletion of `apps/bondy/doc` build (-120 files from 20db1a5)

I don't think I should touch any file from a merge except for the conflicts.
That's calling for trouble..

**Comment by alan-ls** [comment]
Close it as @alejandro-miguez will do it directly on this repository

**Pull request closed without merging**